adc/consumer.py

```diff
@@ -25,6 +25,7 @@ def __init__(self, conf: 'ConsumerConfig') -> None:
         # Workaround for https://github.com/edenhill/librdkafka/issues/3263.
         # Remove once confluent-kafka-python 1.9.0 has been released.
         self._consumer.poll(0)
+        self.running = False
```

Comment:
I think this would be better as a private variable.

Reply:
I agree, and furthermore I think it should probably be a [`threading.Event`](https://docs.python.org/3/library/threading.html#event-objects) object, instead of a simple boolean. Practically speaking, using a boolean will work fine in CPython as long as the GIL prevents threads from actually executing different parts of this code concurrently, but for compatibility with other implementations and future-proofing, I think a thread-safe object should be used.